### PR TITLE
Fix several binary payment options behavior

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When a payment module was returning several binary payment options, it wasn't working. This PR fixes it. |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | You can install the following module (https://github.com/maximebiloe/bankwire) on the `atos` branch. |

### CODE REVIEW

File: controllers/admin/AdminStoresController.php [hunk 1]
```diff
@@ -535,6 +535,7 @@ public function updateOptionPsShopCountryId($value)
                 Configuration::updateValue('PS_SHOP_COUNTRY_ID', $value);
                 Configuration::updateValue('PS_SHOP_COUNTRY', pSQL($country->name));
             }
+            Hook::exec('actionUpdateStoresAfter');
```
Comment (kpodemski) [inline controllers/admin/AdminStoresController.php]: ooops

File: install-dev/data/xml/hook.xml [hunk 2]
```diff
@@ -357,5 +357,8 @@
     <hook id="actionValidateCustomerAddressForm">
       <name>actionValidateCustomerAddressForm</name><title>Customer address form validation</title><description>This hook is called when a customer submit its address form</description>
     </hook>
+    <hook id="actionUpdateStoresAfter">
```
Comment (kpodemski) [inline install-dev/data/xml/hook.xml]: ooops

File: install-dev/upgrade/sql/1.7.0.0.sql [hunk 3]
```diff
@@ -160,3 +160,4 @@ INSERT INTO  `PREFIX_configuration` (`id_configuration` ,`id_shop_group` ,`id_sh
 DELETE FROM `PREFIX_configuration` WHERE `name` IN ('PS_STORES_DISPLAY_FOOTER', 'PS_STORES_SIMPLIFIED', 'PS_STORES_CENTER_LAT', 'PS_STORES_CENTER_LONG', 'PS_STORES_DISPLAY_SITEMAP');
 
 INSERT INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES ('actionValidateCustomerAddressForm', 'Customer address form validation', 'This hook is called when a customer submit its address form', '1');
+INSERT INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES ('actionUpdateStoresAfter', 'Stores update', 'This hook is called after the stores update', '1');
```
Comment (kpodemski) [inline install-dev/upgrade/sql/1.7.0.0.sql]: ooops

**Comment by marcpicaud** [comment]
Tested, and everything is ok ! Thank you @maximebiloe  💪

**Pull request merged**